webapp/client/src/commonMain/kotlin/models/GetMyRolesRequestHandler.kt

```diff
@@ -18,7 +18,7 @@ class GetMyRolesRequestHandler(
 ) : RequestHandler {
     override suspend fun ableToHandle(request: BaseRequest<*>): Boolean = request is GetMyRolesRequest
 
-    override suspend fun handle(userId: UserId, request: BaseRequest<*>): HandlingResult<GetMyRolesRequest.Response?> {
+    override suspend fun handle(userId: UserId, request: BaseRequest<*>): HandlingResult<*> {
```

Comment:
Is this change required? It looks like it do not change anything but allow return from handler anything instead of some presetup type